docs/application/dotnet/guides/nui/window.md

```diff
@@ -0,0 +1,289 @@
+
+The supported orientation are `Portrait`, `Landscape`, `PortraitInverse` and `LandscapeInverse`.
+`Portrait` means the height is larger than the width or same.
+`Landscape` means the width is larger than the height.
+
+<img src="./media/device_rotate2.png">
+
+
+Application can enable or disable a specific orientation to use orientation functions. The following code shows that `Portrait`, `Landscape`, `PortraitInverse` and `LandscapeInverse` are supported in the application.
+If the device's width is larger than the height, then the window will be shown as landscape when application is launched. If the device's width is smaller than the height or same, then the window will be shown as portrait.
+```csharp
+// To enable Portrait
+window.AddAvailableOrientation( Portrait );
+
+// To enable Landscape
+window.AddAvailableOrientation( Landscape );
+
+// To disable PortraitInverse
+window.RemoveAvailableOrientation( PortraitInverse );
+
+// To enable LandscapeInverse
+window.AddAvailableOrientation( LandscapeInverse );
+```
+
+
+### Preferred orientation
+
+If you want the window to appear in a specific orientation, the preferred orientation function is supported. The preferred orientation is one of the available orientations. For example, as shown in the following code, the available orientation list has `Portrait`, `Landscape`, and `LandscapeInverse`, and the preferred orientation is set to Landscape. This implies that the window will appear in landscape even when the device is rotated with any other orientation.
+
+```csharp
+// To enable available orientations
+window.AddAvailableOrientation( Portrait );
+window.AddAvailableOrientation( Landscape );
+window.AddAvailableOrientation( LandscapeInverse );
+
+// To set the preferred orientation with Landscape.
+window.SetPreferredOrientation( Landscape );
+```                                                                                                                               
+> [!NOTE]
+>
+> To use `SetPreferredOrientation()`, you must first list up the supported orientation using `AddAvailableOrientation()`. Otherwise, the rotation does not work as expected.
+>
+
+## Multi-Window
+MultiWIndow means that application use additional window except default window.
+
+<img src="./media/multiwindow3.png">
+
+To Create MultiWindow, the device must support the following features:
+- https://www.tizen.org/feature/opengles.surfaceless_context
+
+### Create additional window
+
+It is easy to create an additional window in addition to the default window. The size and position of the new window should be specified. Each window can have its own background color and title:
+
+```csharp
+Window newWindow = new Window(new Rectangle(0, 0, 1920, 1080))
+{
+    BackgroundColor = Color.White,
+    Title = "new window"
+};
+```
+
+> **Note**
+>
+> To support rotation in multi window, the rotation functions should be used for each window.
+>
```

Comment:
Please use markdown tag

and suggest changing to:

To support rotation in multi-window, the [rotation](#rotation) function must be used for each window.

Reference: https://github.com/Samsung/tizen-docs/pull/1170#discussion_r451808564

Reply:
@surbhig12 Would you please check the link in your suggestion also? For now, the \[rotation](#rotation) is going to nowhere.

So the hyperlink is suggested to be removed, or changed to use actual anchor.